### ISSUE TITLE
namespace error with AEM content fragment

### ISSUE THREAD

**Issue by KennyTaylor** [oozcitak/xmlbuilder2#37]
this code gives the following error

```
template = {
  'jcr:root': {
    '@/whoops/xmlns:jcr': 'http://www.jcp.org/jcr/1.0',
  }
}
const cf = lodash.cloneDeep(template);
lodash.set(cf, 'jcr:root.jcr:content.data.master.@/whoops/id', '1234');
const doc = create(cf);
let xml = doc.end({ prettyPrint: true });
```
 (I added /whoops/ so I didn't tag those people in this)
```
NamespaceError: The operation is not allowed by Namespaces in XML. [XMLNS] Qualified name includes a prefix but the namespace is null.
```
I believe this error should not be present, or at least should not stop the program. If this is incorrect xml namespace syntax, it is still the xml syntax that AEM is expecting to see, as it follows exactly the file I downloaded from AEM. Maybe this error should be possible to disable, like an "AEM mode" 

**Version:**
 - node.js: 12.17.0
 - xmlbuilder2 2.1.7

**Comment by oozcitak** [comment]
I just added a test and it passes. Can you recheck please?

**Comment by someyoungideas** [comment]
I just ran into this same error message, but I am not completely sure it is related. Nonetheless I was able to pinpoint this issue when adding a `xmlns:soap12="http://www.w3.org/2003/05/soap-envelope"` attribute to to an element. It looks like it may have something to do with the number in the attribute key. I am not confident in that, but it came to mind because if I modify the attribute to `xmlns:soap="http://www.w3.org/2003/05/soap-envelope"` it is successful.

My use case was creating a document from an XML file, formatting it to an object, and then formatting that back to xml. Let me know if I can help with anything else to replicate.
